tests/roles/backend_services/tasks/main.yaml

```diff
@@ -83,7 +83,8 @@
 
       ovs:
         enabled: false
-        template: {}
+        template:
+          external-ids: {}
```

Comment:
just tested in my environment and confirmed works to get passed this step

```
[zuul-worker@np0003756663 data-plane-adoption]$ git diff
diff --git a/tests/roles/backend_services/tasks/main.yaml b/tests/roles/backend_services/tasks/main.yaml
index 476f557..f550236 100644
--- a/tests/roles/backend_services/tasks/main.yaml
+++ b/tests/roles/backend_services/tasks/main.yaml
@@ -83,7 +83,8 @@
 
       ovs:
         enabled: false
-        template: {}
+        template:
+          external-ids: {}
 
       neutron:
         enabled: false


TASK [backend_services : deploy backend services] ******************************
changed: [localhost] => {"changed": true, "cmd": "set -euxo pipefail\n\n\noc apply -f - <<EOF\napiVersion: core.openstack.org/v1beta1\nkind: OpenStackControlPlane\nmetadata:\n  name: openstack\nspec:\n  secret: osp-secret\n  storageClass: local-storage\n\n  keystone:\n    enabled: false\n    template: {}\n\n  mariadb:\n    templates:\n      openstack:\n        containerImage: quay.io/tripleozedcentos9/openstack-mariadb:current-tripleo\n        storageRequest: 500M\n\n  rabbitmq:\n    templates:\n      rabbitmq:\n        replicas: 1\n      rabbitmq-cell1:\n        replicas: 1\n\n  placement:\n    enabled: false\n    template: {}\n\n  glance:\n    enabled: false\n    template:\n      glanceAPIInternal: {}\n      glanceAPIExternal: {}\n\n  cinder:\n    enabled: false\n    template:\n      cinderVolumes: {}\n\n  ovn:\n    enabled: false\n    template: {}\n\n  ovs:\n    enabled: false\n    template:\n      external-ids: {}\n\n  neutron:\n    enabled: false\n    template: {}\n\n  nova:\n    enabled: false\n    template: {}\n\n  ironic:\n    enabled: false\n    template:\n      ironicConductors: []\n\n  manila:\n    enabled: false\n    template:\n        manilaAPI: {}\n        manilaScheduler: {}\n        manilaShares: {}\nEOF\n", "delta": "0:00:00.759544", "end": "2023-04-20 08:30:39.925895", "msg": "", "rc": 0, "start": "2023-04-20 08:30:39.166351", "stderr": "+ oc apply -f -", "stderr_lines": ["+ oc apply -f -"], "stdout": "openstackcontrolplane.core.openstack.org/openstack created", "stdout_lines": ["openstackcontrolplane.core.openstack.org/openstack created"]}

TASK [backend_services : wait for mariadb to start up] *************************


```

Reply:
Thanks :)